tycho-its/src/test/java/org/eclipse/tycho/test/compiler/MavenCompilerPluginTest.java

```diff
@@ -40,6 +40,15 @@ public void testAdditionalBundles() throws Exception {
         verifier.verifyErrorFreeLog();
     }
 
+
+    @Test
+    public void testAdditionalBundles2() throws Exception {
+        Verifier verifier = getVerifier("compiler.additional.bundles2", true);
```

Comment:
```suggestion
        Verifier verifier = getVerifier("compiler.additional.bundles2", false, false);
```

Should be more suitable here, then you automatically ignore local artifacts.